--- ./flowblade-trunk/flowblade.orig	2019-12-13 09:18:10.874337994 +0100
+++ ./flowblade-trunk/flowblade	2019-12-13 09:21:34.887354575 +0100
@@ -36,7 +36,8 @@
 # When running on distro.
 if launch_dir == "/usr/bin":
     print ("Running from installation...")
-    modules_path = "/usr/share/flowblade/Flowblade"
+    from distutils.sysconfig import get_python_lib
+    modules_path = "%s/Flowblade" % get_python_lib()
     if not os.path.isdir(modules_path):
         modules_path = "/usr/share/pyshared/Flowblade"
     print ("modules path:", modules_path)
@@ -46,11 +47,11 @@
     modules_path = "/app/share/flowblade/Flowblade"
     if not os.path.isdir(modules_path):
         modules_path = "/app/share/pyshared/Flowblade"
-    print ("modules path:", modules_path)
 # When running in filesystem.
 else:
     print ("Running from filesystem...")
     modules_path = launch_dir + "/Flowblade"
+    print ("modules path:", modules_path)
 
 sys.path.insert(0, modules_path)
 
