--- ./flowblade	2015-06-16 22:25:18.000000000 +0100
+++ ./flowblade	2015-06-21 06:58:24.782414252 +0100
@@ -34,13 +34,14 @@ print "Launch script dir:", launch_dir
 # Update sys.path to include modules
 if launch_dir == "/usr/bin":
     print "Running from installation..."
-    modules_path = "/usr/share/flowblade/Flowblade"
+    from distutils.sysconfig import get_python_lib
+    modules_path = "%s/Flowblade" % get_python_lib()
     if not os.path.isdir(modules_path):
         modules_path = "/usr/share/pyshared/Flowblade"
-    print "modules path:", modules_path
 else:
     print "Running from filesystem..."
     modules_path = launch_dir + "/Flowblade"
+print "modules path:", modules_path
 
 sys.path.insert(0, modules_path)
 sys.path.insert(0, modules_path + "/vieweditor")
