diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/flowblade flowblade-2.8.0.3/flowblade-trunk/flowblade
--- flowblade-2.8.0.3.orig/flowblade-trunk/flowblade	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/flowblade	2022-01-13 23:21:50.320285722 +0000
@@ -57,7 +57,7 @@ sys.path.insert(0, modules_path)
 
 # Check that we have MLT, missing is fatal.
 try:
-    import mlt
+    import mlt7 as mlt
     try:
         mlt_version = mlt.LIBMLT_VERSION
         print ("MLT found, version:", mlt_version)
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/app.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/app.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/app.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/app.py	2022-01-13 23:22:06.042219189 +0000
@@ -40,7 +40,7 @@ from gi.repository import Gtk
 from gi.repository import Gdk
 
 import locale
-import mlt
+import mlt7 as mlt
 import hashlib
 import os
 import sys
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiomonitoring.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiomonitoring.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiomonitoring.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiomonitoring.py	2022-01-13 23:22:06.043219185 +0000
@@ -29,7 +29,7 @@ except ImportError:
 import gi
 
 import cairo
-import mlt
+import mlt7 as mlt
 import time
 
 from gi.repository import Gtk, GObject
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiosync.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiosync.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiosync.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiosync.py	2022-01-13 23:22:06.043219185 +0000
@@ -23,7 +23,7 @@ Handles syncing clips based on their aud
 """
 import datetime
 import hashlib
-import mlt
+import mlt7 as mlt
 import os
 import subprocess
 import sys
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiowaveform.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiowaveform.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiowaveform.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiowaveform.py	2022-01-13 23:22:06.043219185 +0000
@@ -22,7 +22,7 @@
 Modules handles creating and caching audio waveform images for clips.
 """
 
-import mlt
+import mlt7 as mlt
 import os
 import pickle
 import threading
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiowaveformrenderer.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiowaveformrenderer.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/audiowaveformrenderer.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/audiowaveformrenderer.py	2022-01-13 23:22:06.044219181 +0000
@@ -23,7 +23,7 @@ Modules handles creating and caching aud
 """
 
 import locale
-import mlt
+import mlt7 as mlt
 import os
 import pickle
 import subprocess
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/clipmenuaction.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/clipmenuaction.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/clipmenuaction.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/clipmenuaction.py	2022-01-13 23:22:06.048219164 +0000
@@ -27,7 +27,7 @@ from PIL import Image
 from gi.repository import GLib
 from gi.repository import Gtk
 
-import mlt
+import mlt7 as mlt
 from operator import itemgetter
 import os
 import shutil
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/containeractions.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/containeractions.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/containeractions.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/containeractions.py	2022-01-13 23:22:06.049219159 +0000
@@ -24,7 +24,7 @@ from gi.repository import GLib
 import cairo
 import copy
 import hashlib
-import mlt
+import mlt7 as mlt
 import os
 from os import listdir
 from os.path import isfile, join
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/exporting.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/exporting.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/exporting.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/exporting.py	2022-01-13 23:22:06.108218910 +0000
@@ -22,7 +22,7 @@ from gi.repository import Gtk
 import os
 from xml.dom import minidom
 from math import floor
-import mlt
+import mlt7 as mlt
 import time
 import hashlib
 import re
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/medialinker.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/medialinker.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/medialinker.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/medialinker.py	2022-01-13 23:22:06.145218753 +0000
@@ -19,7 +19,7 @@
 """
 
 import glob
-import mlt
+import mlt7 as mlt
 import locale
 import os
 import subprocess
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltenv.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltenv.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltenv.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltenv.py	2022-01-13 23:22:06.148218740 +0000
@@ -22,7 +22,7 @@
 Module checks environment for available codecs and formats.
 """
 from gi.repository import GObject
-import mlt
+import mlt7 as mlt
 
 import dialogutils
 import editorstate
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltfilters.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltfilters.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltfilters.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltfilters.py	2022-01-13 23:22:06.149218736 +0000
@@ -27,7 +27,7 @@ import copy
 
 from gi.repository import GdkPixbuf
 
-import mlt
+import mlt7 as mlt
 import xml.dom.minidom
 
 import appconsts
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltplayer.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltplayer.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltplayer.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltplayer.py	2022-01-13 23:22:06.149218736 +0000
@@ -24,7 +24,7 @@ a Xwindow of a GTK+ widget and os audios
 """
 from gi.repository import Gdk
 
-import mlt
+import mlt7 as mlt
 import os
 import time
 
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltprofiles.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltprofiles.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mltprofiles.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/mltprofiles.py	2022-01-13 23:22:06.150218732 +0000
@@ -22,7 +22,7 @@
 MLT framework profiles.
 """
 import os
-import mlt
+import mlt7 as mlt
 import xml.dom.minidom
 
 import appconsts
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mlttransitions.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/mlttransitions.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/mlttransitions.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/mlttransitions.py	2022-01-13 23:22:06.151218728 +0000
@@ -23,7 +23,7 @@ Module contains objects that wrap mlt.Tr
 two tracks.
 """
 import copy
-import mlt
+import mlt7 as mlt
 import os
 import xml.dom.minidom
 
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/monitorwidget.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/monitorwidget.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/monitorwidget.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/monitorwidget.py	2022-01-13 23:22:06.208218487 +0000
@@ -21,7 +21,7 @@ from gi.repository import Gdk
 from gi.repository import Gtk, GLib
 
 import cairo
-import mlt
+import mlt7 as mlt
 import numpy as np
 import os
 import threading
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/patternproducer.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/patternproducer.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/patternproducer.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/patternproducer.py	2022-01-13 23:22:06.211218474 +0000
@@ -27,7 +27,7 @@ import hashlib
 
 from gi.repository import Gtk, Gdk
 
-import mlt
+import mlt7 as mlt
 
 import appconsts
 import dialogutils
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/projectaction.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/projectaction.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/projectaction.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/projectaction.py	2022-01-13 23:22:06.216218453 +0000
@@ -28,7 +28,7 @@ import datetime
 import fnmatch
 import glob
 import hashlib
-import mlt
+import mlt7 as mlt
 import os
 from os import listdir
 from os.path import isfile, join, expanduser
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/projectdata.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/projectdata.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/projectdata.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/projectdata.py	2022-01-13 23:22:06.217218448 +0000
@@ -23,7 +23,7 @@ Module contains objects used to capture
 """
 import cairo
 import datetime
-import mlt
+import mlt7 as mlt
 import hashlib
 import os
 
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/projectmediaimport.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/projectmediaimport.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/projectmediaimport.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/projectmediaimport.py	2022-01-13 23:22:06.252218300 +0000
@@ -18,7 +18,7 @@
     along with Flowblade Movie Editor.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-import mlt
+import mlt7 as mlt
 import locale
 import os
 import subprocess
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/proxyediting.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/proxyediting.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/proxyediting.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/proxyediting.py	2022-01-13 23:22:06.256218283 +0000
@@ -20,7 +20,7 @@
 
 import glob
 import hashlib
-import mlt
+import mlt7 as mlt
 import os
 import shutil
 import threading
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/renderconsumer.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/renderconsumer.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/renderconsumer.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/renderconsumer.py	2022-01-13 23:22:06.257218279 +0000
@@ -25,7 +25,7 @@ Module contains objects and methods need
 
 from gi.repository import Gdk
 
-import mlt
+import mlt7 as mlt
 import time
 import threading
 import xml.dom.minidom
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/render.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/render.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/render.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/render.py	2022-01-13 23:22:06.258218275 +0000
@@ -28,7 +28,7 @@ Rendering is done in app.player object o
 
 from gi.repository import Gtk
 
-import mlt
+import mlt7 as mlt
 import hashlib
 import os
 import subprocess
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/sequence.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/sequence.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/sequence.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/sequence.py	2022-01-13 23:22:06.260218266 +0000
@@ -23,7 +23,7 @@ Module contains class Sequence that is t
 by the application. A project has 1-n of these.
 """
 import time
-import mlt
+import mlt7 as mlt
 import os
 
 import appconsts
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tlinerenderserver.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tlinerenderserver.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tlinerenderserver.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tlinerenderserver.py	2022-01-13 23:22:06.265218245 +0000
@@ -32,7 +32,7 @@ import dbus
 import dbus.service
 from dbus.mainloop.glib import DBusGMainLoop
 import locale
-import mlt
+import mlt7 as mlt
 import os
 import subprocess
 import sys
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/batchrendering.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/batchrendering.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/batchrendering.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/batchrendering.py	2022-01-13 23:22:13.960185679 +0000
@@ -29,7 +29,7 @@ from gi.repository import Gtk, Gdk, GdkP
 import dbus
 import dbus.service
 from dbus.mainloop.glib import DBusGMainLoop
-import mlt
+import mlt7 as mlt
 import hashlib
 import locale
 import os
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/blenderheadless.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/blenderheadless.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/blenderheadless.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/blenderheadless.py	2022-01-13 23:22:13.960185679 +0000
@@ -30,7 +30,7 @@ gi.require_version('Gtk', '3.0')
 from gi.repository import GLib
 
 import locale
-import mlt
+import mlt7 as mlt
 import os
 from os import listdir
 from os.path import isfile, join
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/gmicheadless.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/gmicheadless.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/gmicheadless.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/gmicheadless.py	2022-01-13 23:22:13.962185671 +0000
@@ -30,7 +30,7 @@ gi.require_version('Gtk', '3.0')
 from gi.repository import GLib
 
 import locale
-import mlt
+import mlt7 as mlt
 import os
 import pickle
 import subprocess
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/gmicplayer.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/gmicplayer.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/gmicplayer.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/gmicplayer.py	2022-01-13 23:22:13.965185658 +0000
@@ -23,7 +23,7 @@ Clip player used to select frames for pr
 """
 
 
-import mlt
+import mlt7 as mlt
 import os
 from os import listdir
 from os.path import isfile, join
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/gmic.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/gmic.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/gmic.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/gmic.py	2022-01-13 23:22:13.966185654 +0000
@@ -29,7 +29,7 @@ from gi.repository import Pango
 
 import cairo
 import locale
-import mlt
+import mlt7 as mlt
 import os
 import re
 import shutil
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/mltheadlessutils.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/mltheadlessutils.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/mltheadlessutils.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/mltheadlessutils.py	2022-01-13 23:22:13.966185654 +0000
@@ -24,7 +24,7 @@ in initialized Flowblade/MLT enviroment.
 """
 
 import locale
-import mlt
+import mlt7 as mlt
 import os
 
 import ccrutils
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/mltxmlheadless.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/mltxmlheadless.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/mltxmlheadless.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/mltxmlheadless.py	2022-01-13 23:22:13.966185654 +0000
@@ -30,7 +30,7 @@ gi.require_version('Gtk', '3.0')
 from gi.repository import GLib
 
 import locale
-import mlt
+import mlt7 as mlt
 import os
 import pickle
 import subprocess
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/motionheadless.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/motionheadless.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/motionheadless.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/motionheadless.py	2022-01-13 23:22:13.966185654 +0000
@@ -18,7 +18,7 @@
     along with Flowblade Movie Editor. If not, see <http://www.gnu.org/licenses/>.
 """
 
-import mlt
+import mlt7 as mlt
 import os
 import threading
 import time
diff -rup flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/proxyheadless.py flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/proxyheadless.py
--- flowblade-2.8.0.3.orig/flowblade-trunk/Flowblade/tools/proxyheadless.py	2021-03-30 17:41:58.000000000 +0100
+++ flowblade-2.8.0.3/flowblade-trunk/Flowblade/tools/proxyheadless.py	2022-01-13 23:22:13.967185650 +0000
@@ -19,7 +19,7 @@
 """
 
 import glob
-import mlt
+import mlt7 as mlt
 import os
 from PIL import Image
 import threading
